docs/providers/aws/guide/serverless.yml.md

```diff
@@ -24,6 +24,7 @@ service: myService
 frameworkVersion: '2'
 enableLocalInstallationFallback: false # If set to 'true', guarantees that it's a locally (for service, in its node_modules) installed framework which processes the command
 useDotenv: false # If set to 'true', environment variables will be automatically loaded from .env files
+throwOnUnresolvedVariableReferences: false # If set to 'true', references to variables that cannot be resolved will result in an error being thrown
```

Comment:
I think I would prefer sticking to `unresolvedVariablesNotificationMode` as enum as that gives us a bit more flexibility in the future if needed (if there would be a request to introduce another mode, or e.g. turning warnings/errors completely)